src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -143,19 +150,101 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
         'West Europe',
         'West US 2'
     ];
-
     const webProvider: string = 'Microsoft.Web';
-
     LocationListStep.setLocationSubset(wizardContext, Promise.resolve(locations), webProvider);
     LocationListStep.addStep(wizardContext, promptSteps, {
         placeHolder: localize('selectLocation', 'Select a region for Azure Functions API and staging environments'),
         noPicksMessage: localize('noRegions', 'No available regions.')
     });
-
     promptSteps.push(new BuildPresetListStep(), new AppLocationStep(), new ApiLocationStep(), new OutputLocationStep());
-
     executeSteps.push(new VerifyProvidersStep([webProvider]));
     executeSteps.push(new StaticWebAppCreateStep());
+    const wizard: AzureWizard<IStaticWebAppWizardContext> = new AzureWizard(wizardContext, {
+        title,
```

Comment:
can we extract certain parts into functions to make the code more readable please

Reply:
Or section off the code into different blocks